tools/e2etesting/TestEventProcessor/TestEventProcessor.Businesslogic/StopResult.cs

```diff
@@ -68,5 +68,20 @@ public class StopResult : IResult
         /// Indicates the number of duplicate messages detected by IncrementalIntValueChecker.
         /// </summary>
         public uint DuplicateValueCount { get; set; }
+
+        /// <summary>
+        /// Indicates the number of dropped sequence numbers detected by SequenceNumberChecker.
+        /// </summary>
+        public uint DroppedSequenceCount { get; set; }
```

Comment:
Can you please add those fields into `StopResult` model that we have in E2E project as well ?

Reply:
done

Reply:
Here is the file path for that one: `e2e-tests\IIoTPlatform-E2E-Tests\TestEventProcessor\StopResult.cs`